scrape_your_plate.py

```diff
@@ -158,20 +159,23 @@ def save_recipe(title, id, soup):
     parser = argparse.ArgumentParser(description='Scrape recipies from Pepperplate')
     parser.add_argument('username', help='Username to log in with')
     parser.add_argument('password', nargs="?", default=None, help='Password to log in with. If not provided on the command line it will be requested by the program')
+    parser.add_argument('directory', nargs="?", default='recipes', help='Directory to which download everything. defaults to "recipes"')
     args = parser.parse_args()
 
+    imgpath = os.path.join(args.directory, 'img')
+    if not os.path.exists(imgpath):
```

Comment:
Good idea. I think it would be good to check and create the save directory at the same time. Or at least trap and error gracefully.